OutOfSchool/OutOfSchool.WebApi.Tests/Services/ProviderServicesTests/PublicProviderServiceTests.cs

```diff
@@ -88,5 +92,41 @@ public async Task UpdateStatus_WhenProviderIdIsNonExistent_ReturnsNull()
         Assert.IsNull(result);
     }
 
+    [Test]
+    public async Task UpdateStatus_WhenDtoIsValid_UpdateAddedInChangesLog()
+    {
+        // Arrange
+        var expectedCallTimes = Times.Once;
```

Comment:
(Readability) This `expectCallTimes` is needed only in last line of the test and olny for assert section. Please consider moving it theer or even inlining it

Reply:
Fixed